spec.html

```diff
@@ -352,7 +358,13 @@ <h3>Notes</h3>
       type uses delta and delta-of-delta compression and
       run-length-encoding for efficient element storage. Also has an
       encoding for sparse arrays containing missing values.</li>
-    <li>The BSON "binary" or "BinData" datatype is used to represent
+    <li>Vector - A densely packed array of numbers, all of the same
```

Comment:
we should probably detail which element types are supported, as detailed in the [spec](https://docs.google.com/document/d/17G17BlfMhtg3QoZHx5Ol0uvLonX0gwJgPCwZl7YUycI/edit#heading=h.2ji6i320cghn). This way users looking to implement this on their own ahead of driver helper support have a path to follow, similar to the following example in Python:
```
def generate_bson_vector(vector):
  buf : typing.List = []
  VECTOR_SUBTYPE = 9
  type_kind = 2
  type_size = 7
  buf.append((type_kind << 4) | type_size) # float32 type_byte
  buf.append(0) # padding byte
  #vector = np.ones(vector_len, dtype=np.float32)
  buf.extend(vector.tobytes("C"))
  return binary.Binary(bytearray(buf), subtype=VECTOR_SUBTYPE)
```

Reply:
if you're going to link to docs, i suggest linking here: https://www.mongodb.com/docs/atlas/atlas-vector-search/create-bson-vectors/

Reply:
sorry i think i was looking at a stale commit, feel free to resolve